plugins/janus_sip.c

```diff
@@ -6653,15 +6795,14 @@ gpointer janus_sip_sofia_thread(gpointer user_data) {
 		return NULL;
 	}
 	JANUS_LOG(LOG_VERB, "Joining sofia loop thread (%s)...\n", session->account.username);
-	session->stack = g_malloc0(sizeof(ssip_t));
-	su_home_init(session->stack->s_home);
```

Comment:
Why have you removed this? AFAIK it's needed.

Reply:
stack is always created in janus_sip_create_session.